pkgs/development/libraries/cgreen/default.nix

```
@@ -11,6 +11,10 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-aQrfsiPuNrEMscZrOoONiN665KlNmnOiYj9ZIyzW304=";
   };
 
+  postPatch = ''
+    grep -rIl '/usr/bin/nm' . | xargs sed -i 's#/usr/bin/nm#nm#g'
```

```suggestion
    substituteInPlace tools/tests/discoverer_acceptance_tests.c \
      --replace "/usr/bin/nm" "nm"
```
Where is nm coming from? Can we hardcode the path to the package?

---

It comes from `bintools` which is provided by `stdenv` and therefore is always present when building derivations. In addition, the `nm` that's used needs to be the one that fits with the build tools being used by the derivation that's using the `cgreen` library, and not the tools used to build `cgreen` itself, so I think it works be inappropriate to hardcode the path here.